bson.cpp

```diff
@@ -386,8 +385,7 @@ void VariantToBsonConverter::_convertSerializable(bson_t *bson, const char *key,
 		result.asTypedValue(),
 		m,
 		v.get(),
-		nullptr,
-		1, args
+		nullptr
```

Comment:
Yes, I was needlessly passing it in. This issue came up when calling the internal native bsonSerialize (really __debugInfo) method. Apparently the warning didn't trigger for userland implementations, as I certainly never saw it and there are tests.